Dockerfile

```diff
@@ -61,13 +63,24 @@ RUN go get -d github.com/mendersoftware/mender
 WORKDIR $GOPATH/src/github.com/mendersoftware/mender
 RUN git checkout $MENDER_CLIENT_VERSION
 
-ENV CC "arm-buildroot-linux-gnueabihf-gcc"
+# Toolchain configuration
+ARG toolchain_host
+ENV TOOLCHAIN_HOST ${toolchain_host:-arm-buildroot-linux-gnueabihf}
```

Comment:
I don't know this syntax; looks like inline if statement to assign var, similar to C?).
Just wanted to make sure that the - in front of arm-... is right?

Reply:
This is bash - Parameter Substitution :), and yeah the `-` should be there. 

But based on you second comment I will move this to the build script instead and make these arguments mandatory.